### PR TITLE
Add Delete Button and MongoDB Atlas Deploy

### PR DESCRIPTION
**Pull request by Pia0912**
Feature UserStory 5
- add delete button and delete methods in backend
- add Tests for Delete Button
- add unit and integration tests for WebMvcConfig
- add mock static folder with test resources

### CODE REVIEW

File: backend/src/test/resources/static/existing-resource.txt
Comment (Kajochi) [inline backend/src/test/resources/static/existing-resource.txt:1]: What for a test is this?

Comment (toshydev) [reply backend/src/test/resources/static/existing-resource.txt:1]: This is a test resource for the WebMvcConfig test. The mocked MVC requests an existing resource and should answer with the content of the file and status 200.

**Review by Kajochi — APPROVED** [review]
LGTUs 
ps: on some places you use inlineStyling. If it is ok for you, it´s ok for us. But think about i;)

**Comment by toshydev** [comment]
> LGTUs ps: on some places you use inlineStyling. If it is ok for you, it´s ok for us. But think about i;)

I know right? Inline styling = ugly af xD 
But we'll work on the styling issues in the next user stories 💪

**Pull request merged**